src/renderers/webgl/WebGLTextures.js

```diff
@@ -9,6 +9,8 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 	const maxCubemapSize = capabilities.maxCubemapSize;
 	const maxTextureSize = capabilities.maxTextureSize;
 	const maxSamples = capabilities.maxSamples;
+	const msaartcSupported = extensions.has( 'EXT_multisampled_render_to_texture' );
```

Comment:
What does `rtc` stands for in `msaartcSupported`?

Reply:
MultisampledRenderToTexture